k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -110,6 +111,7 @@
  * it removes itself. Thus, any commands that that activity submitted are
  * removed from the queue once the activity is no longer active.
  */
+@SuppressWarnings("unchecked") // TODO change architecture to actually work with generics
```

Comment:
If you're annoyed by the warnings either disable the inspection on your machine or fix the code. Don't suppress the warnings for everyone.